ticketguardian/abstract/patch_resource.py

```diff
@@ -0,0 +1,31 @@
+import requests
+
+from ticketguardian.abstract.api_resource import APIResource
+from ticketguardian.abstract.error_handling import raise_response_error
+
+
+class PatchResourceMixin(APIResource):
+
+    def patch(self, request, *args, **kwargs):
+        """
+        Patch a resource with kwargs.
+            Keyword Arguments:
+                ext: Strings that are extensions of the url
+                     This should only be used from within resource methods.
+                raw_data (bool): A boolean value that will tell this method to
+                                 return the raw list data.
+        """
+        url = self._make_url(resource_id, *ext)
+        raw_data = params.pop('raw_data', False)
```

Comment:
I don't think we will need to support `raw_data` in this mixin. I should have probably explained this better in the other mixins but I had to use `raw_data` only for special cases. If you look in the sdk there are only like 4 instances of it being used.